gollum.gemspec

```diff
@@ -23,18 +23,18 @@ Gem::Specification.new do |s|
   s.rdoc_options = ['--charset=UTF-8']
   s.extra_rdoc_files = %w[README.md LICENSE]
 
-  s.add_dependency 'gollum-lib', '~> 5.0'
+#  s.add_dependency 'gollum-lib', '~> 5.0'
   s.add_dependency 'kramdown', '~> 2.3'
   s.add_dependency 'kramdown-parser-gfm', '~> 1.1.0'
   s.add_dependency 'sinatra', '~> 2.0'
   s.add_dependency 'sinatra-contrib', '~> 2.0'
-  s.add_dependency 'mustache', ['>= 0.99.5', '< 1.0.0']
+  s.add_dependency 'mustache-sinatra', '~> 1.0'
```

Comment:
Did this gem change names?

Reply:
Not exactly, but the Sinatra helpers that we were using were abstracted to a separated gem (also maintained by the mustache org), which in turn requires `mustache`.

Reply:
👍